packages/lb-annotation/src/core/pointCloud/segmentation.ts

```diff
@@ -32,6 +35,7 @@ class PointCloudSegmentOperation {
   constructor(props: IProps) {
     this.dom = props.dom;
     this.store = props.store;
+    this.checkMode = props.checkMode;
```

Comment:
Don't need to pass  checkMode to segmentation directly. Saving checkMode to the store is a better way.

Reply:
done

Reply:
but i see 'isForbid' in segmentation, don't you think 'isForbid' is the same as 'checkMode' in some way?

Reply:
Yeah, but `isForbid` use `store.forbidOperation`. You can add multiple conditions for evaluation like
```ts
  public get isForbid() {
    return this.forbidOperation || this.store.checkMode;
  }
```